app/models/tickets_edit_person.rb

```diff
@@ -0,0 +1,81 @@
+# frozen_string_literal: true
+
+class TicketsEditPerson < ApplicationRecord
+  self.table_name = "tickets_edit_person"
```

Comment:
Why do you have to override the table name here?

Reply:
Hm, I think you can be more loose on the names. It doesn't necessarily have to contain `Ticket` as prefix.
Something like `EditPersonRequest` will be fine. Or if you really want the "ticket" link, you could do `EditPersonTicket`, which Rails correctly maps to `edit_person_tickets`.

But speaking from our experience with the Groups tables so far, I don't think you have to forcefully include `Ticket` everywhere.

Reply:
Actually, a request to change some of your personal details _is not_ a ticket. The request is _handled via_ your ticket system, but that doesn't make the request itself become a ticket. (Just as an argument in favor of dropping `Ticket`)

Reply:
I don't have a clear/strong preference about how to proceed here. Having to override `table_name` always feels a bit weird, but on the other hand your arguments in favor of the override are also very reasonable.

Fine to keep the `self.table_name` override as is, or go with another name as you see fit.